lib/xcode/install.rb

```diff
@@ -380,7 +380,7 @@ def get_dmg(version, progress = true, url = nil, progress_block = nil, retry_dow
       end
       if ENV.key?('XCODE_INSTALL_CACHE_DIR')
         Pathname.glob(ENV['XCODE_INSTALL_CACHE_DIR'] + '/*').each do |fpath|
-          return fpath if /^xcode_#{version}\.dmg|xip$/ =~ fpath.basename.to_s
+          return fpath if /^xcode_#{version}\.(dmg|xip)$/ =~ fpath.basename.to_s.downcase
```

Comment:
could also change the regexp to have the `i` option at the end to make it case-insensitive